client/components/edit/datastream/DatastreamLicenseContent.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import Button from "@mui/material/Button";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import FormControl from "@mui/material/FormControl";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormLabel from "@mui/material/FormLabel";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import useDatastreamOperation from "../../../hooks/useDatastreamOperation";
+import { useEditorContext } from "../../../context/EditorContext";
+
+const licenseMapping = {
+    ATTRIBUTION: "Attribution",
```

Comment:
The intention is to be able to add licenses purely via configuration, so if we have constants in the config file and then map them to strings in the code, this is less flexible than intended. Can we just have a map of description => url (or vice versa) in the .ini file, or are the constraints of the .ini format too strict to allow that to work?

Reply:
Maybe the solution is as simple as moving this map into the config file as well -- I do see advantages to using tokens for licenses (particularly if we need to add an i18n layer later), but we should find a solution where this isn't hard-coded into the components.

Reply:
I think I'll add it to the catalog call.

Reply:
Sounds reasonable to me.